pkg/kubelet/prober/prober.go

```diff
@@ -200,13 +203,19 @@ func (pb *prober) runProbe(p *api.Probe, pod *api.Pod, status api.PodStatus, con
 		return pb.exec.Probe(pb.newExecInContainer(pod, container, containerID, p.Exec.Command))
 	}
 	if p.HTTPGet != nil {
+		scheme := strings.ToLower(string(p.HTTPGet.Scheme))
```

Comment:
This is antithetical to most of the API.  Is the value in the API a constant (in as much as JSON has them) or a literal?  We have lots of constant-ish values but no literal-ish values.  Constants in the API Start with capitals and follow Go style.

For analogy - the API says "TCP" and "UDP", even though docker wants "tcp" and "udp".

@bgrant0607 is the final arbiter, but he's not at his desk right now, so I can't ask him.  So instead he gets an @ here.